### DISCUSSION TITLE
Crafting recipie for opaque pipes.

### DISCUSSION THREAD

**Discussion by mindcloud69** [RS485/LogisticsPipes#1598] [Ideas]
Having to install upgrades on a large pipe network is extremely tedious and for something like opaque pipes which are not a game play advantage per say. I think there should be a default opaque recipe for basic and unrouted pipes similar to how Thermal Dynamics does it.

**Comment by QORTEC** [comment]
I would agree, there should be a craft-able opaque logistics pipes.

I guess one possible recipe be a opaque upgrade with a logistics pipe under it in a crafting table, this recipe would allow you to apply the upgrade up to a stack at once. 
_[1 opaque upgrade + 1 logistics pipe = 1 opaque logistics pipe]_

Though I think the opaque upgrade in general is a bit on the expansive side, if you agree then a alternative recipe could be placing the opaque upgrade in the centre of the crafting table and surrounding it with (8) logistics pipes. 
_[1 opaque upgrade + 8 logistics pipe = 8 opaque logistics pipe]_

**Comment by mindcloud69** [comment]
I think QORTEC's suggestion is a good one. I would agree with the [1 opaque upgrade + 8 logistics pipe = 8 opaque logistics pipe] recipe at the least. Additionally consider an ingot or paper crafted with a pipe for the opaque upgrade. The reason I would suggest such a cheap recipe is that I don't believe this materially effects game play or balance only performance.

**Comment by davboecki** [comment]
If this ever get's implemented [And I don't currently say it will], This will be a long time from now, because for that we would need to lay a complete new layer of crafting/Item handling to handle the upgrade inside the pipe when it is still inside the player inventory. And additionaly the new pipe model doesn't have the opaque rendering anyway. It only doesn't render the items. So that would have to be worked on, too.

**Comment by mindcloud69** [comment]
How about a machine that could install modules and can be automated. That could work just as well and be a lot more flexible.

**Comment by bziemons** [comment]
@mindcloud69 you are free to add any new ideas to the list: https://github.com/RS485/LogisticsPipes/wiki/Feature-Requests

Please make sure they are well understandable.
